src/topology/algebra/order/monotone_convergence.lean

```diff
@@ -108,6 +109,18 @@ by convert tendsto_at_bot_is_lub h_anti.dual ha.dual
 
 end is_glb
 
+section lattice
+
+/-- This lemma wouldn't make sense for `ι` of type `Prop` because there is no `finset ι` in that
+  case.  -/
```

Comment:
I think you can just drop this comment

Reply:
Yeah, this comment was because there were some places we were trying to use `Sort*` instead of `Type*`, but here that's not possible...